### PR TITLE
00 controllers

### PR DESCRIPTION
**Pull request by RoxaneDrelon**
Added controllers, rating table sql file, rating examples, cors and minor improvements here and there.

### CODE REVIEW

**Review by XSylvainX — DISMISSED** [review]

File: conf.js [hunk 1]
```diff
@@ -1,12 +1,10 @@
...
-const db = mysql.createConnection({
-  host: DB_HOST, // adresse du serveur
-  user: DB_USER, // le nom d'utilisateur
-  password: DB_PASSWORD, // le mot de passe
-  database: DB_DATABASE, // le nom de la base de données
+const db = mysql.createPool({
+  host: process.env.DB_HOST, // adresse du serveur
+  user: process.env.DB_USER, // le nom d'utilisateur
+  password: process.env.DB_PASSWORD, // le mot de passe
+  database: process.env.DB_DATABASE, // le nom de la base de données
```
Comment (LoicBrassart) [inline conf.js]: (These comments don't seem so useful to me, are they ?)

File: package.json [hunk 2]
```diff
@@ -18,8 +18,10 @@
   },
   "homepage": "https://github.com/WildCodeSchool/lille-js-202003-project-literate-guide-backend#readme",
   "dependencies": {
+    "cors": "^2.8.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "mysql": "^2.18.1"
+    "mysql": "^2.18.1",
+    "mysql2": "^2.1.0"
```
Comment (LoicBrassart) [inline package.json]: Please remove packages you don't use anymore

File: server.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+app.use(cors());
+app.use("/capsules", capsules);
 
-app.listen(backendPort, () => {
-  console.log(`API root available on http://localhost:${backendPort}`);
+app.listen(4242, (err) => {
+  if (err) {
+    console.log("Erreur serveur");
+  } else {
+    console.log(`API root available on http://localhost:4242`);
```
Comment (LoicBrassart) [inline server.js]: `localhost:${backendPort}` ;-)

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Please remove useless dependancies

**Review by XSylvainX — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**